core/chains/evm/config/v2/defaults/Arbitrum_Goerli.toml

```diff
@@ -0,0 +1,22 @@
+ChainID = '421613'
+ChainType = 'arbitrum'
+LinkContractAddress = "0x615fBe6372676474d9e6933d310469c9b68e9726"
```

Comment:
This seems same as rinkbey. Are they same?

Reply:
Just a placeholder to placate the config validation while we run tests. The tests overwrite this value with their own contract per-run.

Reply:
Cleared for now, and we can circle back later with the official address when it's ready.